systems/notification/event-notify/pkg/server/event.go

```diff
@@ -488,8 +488,11 @@ func (es *EventToNotifyEventServer) EventNotification(ctx context.Context, e *ep
 		if err != nil {
 			log.Errorf("Failed to store raw message for %s to db. Error %+v", c.Name, err)
 		}
+		dynamicConfig := c
```

Comment:
Why making a copy of c, instead of updating it directly?

Reply:
Addressed.